extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantConfigContext.java

```diff
@@ -14,8 +14,15 @@
      */
     final OidcTenantConfig oidcConfig;
 
+    final boolean ready;
+
     public TenantConfigContext(OidcProvider client, OidcTenantConfig config) {
+        this(client, config, true);
+    }
+
+    public TenantConfigContext(OidcProvider client, OidcTenantConfig config, boolean ready) {
         this.provider = client;
         this.oidcConfig = config;
+        this.ready = ready;
```

Comment:
Where is this 'ready' used?

Reply:
@cescoffier Sure, it is checked [here](https://github.com/quarkusio/quarkus/pull/16060/files#diff-fe86b9ea4400174bff7b67f7eb3ca4fe046443d52cff7637f38a2a1613b32b5fR111). Then `OidcAuthenticationMechanism` will fail the request if the tenant context is null.

Reply:
Sorry, I missed it!